baseplate/context/__init__.py

```diff
@@ -53,3 +53,7 @@ def __init__(self, name, context_factory):
     def on_server_span_created(self, context, server_span):
         context_attr = self.context_factory.make_object_for_context(self.name, server_span)
         setattr(context, self.name, context_attr)
+
+    def on_local_span_created(self, context, span, component_name):
+        context_attr = self.context_factory.make_object_for_context(self.name, span)
```

Comment:
I think with my changes all of the references to `server_span` and `ServerSpan` in the context class definitions should be `Span` since objects for contexts won't just be created for server spans anymore.